rpmlint/checks/SUIDPermissionsCheck.py

```diff
@@ -81,7 +81,7 @@ def _check_post_scriptlets(self, pkg, path, need_verifyscript):
             found = False
             if script:
                 for line in script.split('\n'):
-                    if '/chkstat' in line and path in line:
+                    if re.search(fr'(chkstat|permctl) -n .* {path}', line):
```

Comment:
There's a similar line before, maybe we should just do the same and `re.escape` the path here too.
```suggestion
                    escaped = re.escape(path)
                    if re.search(fr'(chkstat|permctl) -n .* {escaped}', line):
```